qiskit_alice_bob_provider/remote/backend.py

```diff
@@ -115,6 +105,7 @@ def run(self, run_input: QuantumCircuit, **kwargs) -> AliceBobRemoteJob:
                 'Please provide an instance of QuantumCircuit.'
             )
         if self._verbose:
+            display_new_line()
```

Comment:
Instead we create the new display at the beginning of `backend.run()`, which is also more relevant in a term of workflow.